architecture/additional_concepts/authorization.adoc

```diff
@@ -228,17 +228,192 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
 and groups] can be associated with, or _bound_ to, multiple roles at the same
 time.
 
-Cluster administrators can visualize these roles, including a matrix of the
-verbs and resources each are associated using the CLI to
-ifdef::openshift-enterprise,openshift-origin[]
-xref:../../admin_guide/manage_authorization_policy.adoc#viewing-cluster-roles[view
-the cluster roles].
-endif::[]
-ifdef::openshift-dedicated[]
-view the cluster roles.
-endif::[]
-Additional *system:* roles are listed as well, which
-are used for various {product-title} system and component operations.
+Cluster administrators can investigate policy roles using the `oc describe` command, which prints a matrix of the `Verbs`, `Non-Resource URLs`, `Extension`, `Resource Names`,`API Groups`, and `Resources` that apply to the policy role.
+
+----
+$ oc describe clusterrole <role>
+----
+
+The rule verbs apply to the objects listed in that row. For example, if the `create` verb is applied to the `serviceaccounts` resource, the holder is allowed to create service accounts but no other resource unless otherwise specified in the role matrix. +
+The verbs are described in the following table.
+
+.Cluster and Local Policy Verbs
+[cols="3",options="header"]
```

Comment:
for better formatting, we can change this to `[cols="1,1,4",options="header"]`, so the third column gets more space as it has more content than the first two.